deepinv/loss/r2r.py

```diff
@@ -0,0 +1,63 @@
+import torch
+import torch.nn as nn
+
+from deepinv.loss.mc import MCLoss
+
+
+class R2RLoss(nn.Module):
+    r"""
+    Recorrupted-to-Recorrupted (R2R) Loss
+
+    This loss can be used for unsupervised image denoising with unorganized noisy images.
+
+    The loss is designed for the noise model:
+
+    .. math::
+
+        y \sim\mathcal{N}(u,\sigma^2 I) \quad \text{with}\quad u= A(x).
+
+    The loss is computed as:
+
+    .. math::
+
+        \| y^- - AR(y^+) \|_2^2 \quad \text{s.t.} \quad y^+ = y + \alpha z, \quad y^- = y -  z / \alpha
+
+    where :math:`R` is the trainable network, :math:`A` is the forward operator,
+    :math:`y` is the noisy measurement,
+    :math:`z` is the additional Gaussian noise of standard deviation :math:`\eta`,
+    and :math:`\alpha` is a scaling factor.
+
+
+    This loss is statistically equivalent to the supervised loss function defined on noisy/clean image pairs
+    according to authors in https://ieeexplore.ieee.org/document/9577798
+
+    :param float eta: standard deviation of the Gaussian noise used for the perturbation.
+    :param float alpha: scaling factor of the perturbation.
+    """
+
+    def __init__(self, metric=torch.nn.MSELoss(), eta=35, alpha=0.5):
```

Comment:
The library uses images in the range [0,1], so I would choose eta in the range [0,1] instead of [0,255]

Reply:
thanks, fixed in [669bf87]